src/engine/renderer/tr_image.cpp

```diff
@@ -1758,12 +1758,22 @@ static void R_LoadImage( const char **buffer, byte **pic, int *width, int *heigh
 		}
 	}
 
-	// the file isn't there, try again without the extension
-	COM_StripExtension3( token, filename, MAX_QPATH );
-
+	// if the file isn't there, maybe the file path did not have any extension,
+	// and it may be possible the file has a dot in its name that was mistakenly
+	// taken as an extension
 	const char *prefix;
 	int bestLoader = R_FindImageLoader( filename, &prefix );
 
+	if ( *ext && bestLoader == -1 )
+	{
+		// if the file isn't there, maybe its filename does not have extension
```

Comment:
Same here

Reply:
It seems out of place because the way the code works.

In the case of `texture/name.blend.tga` named internally `texture/name.blend`, the `ext` string contains `.blend`, but the name itself does not have extension. Maybe I can add explicit words telling that in this particular case, `ext` is not empty but is not an extension, it's part of the name.

Reply:
In that case (`texture/name.blend` requested when actual file is `texture/name.blend.tga`), it will be found by the first `R_FindImageLoader` call above.

Reply:
Hmm, I have to sleep, I will re-read that later :grin:

Reply:
Well, my mind was probably not very clear at this time, I was talking as if it was about the previous line, yes. I just fixed the comment.